kolena/_experimental/dataset/_dataset.py

```diff
@@ -251,9 +253,12 @@ def register_dataset(
                 validate_dataframe_ids(chunk, id_fields)
                 validated = True
             _upload_dataset_chunk(chunk, load_uuid, id_fields)
+
     request = RegisterRequest(name=name, id_fields=id_fields, uuid=load_uuid)
     response = krequests.post(Path.REGISTER, json=asdict(request))
     krequests.raise_for_status(response)
+    data = from_dict(EntityData, response.json())
+    log.info(f"Successfully created/updated dataset {name} {get_dataset_url(dataset_id=data.id)}")
```

Comment:
Let's keep the link off until the feature is deployed to production.

Reply:
Why?

Reply:
It's not workable now, right?

Reply:
True, but it doesn't feel any more or less broken to me to print success messages with a URL that redirects you away than to print success messages about content you have no way of reaching. In the interest of not forgetting to add URLs in a follow-up prior to release, I'm fine with adding links before this goes live in January.

Reply:
Ack